load_post_run.py

```diff
@@ -200,17 +243,27 @@ def unload_lab():
 load_proc,fdps = find_load_proc()
 print(load_proc + " : " + fdps)
 #Run appropriate test commands for the user input values 1-5
+
+# Suggestion: instead of defining "test" in each elif, map it before (i think they're called python dictories). 
+# Don't need to do elif each time since they will be running the same commands.
 if test_run == 1:
     test = "ZAN Clean Run"
     print(test)
 elif test_run == 2:
     test = "ZOA DIL"
     print(test)
     post_files()
+    load_lab()
+    time.sleep(300)
+    #print("starting FDP dp_comp.exe commands @ XX:XX ...")
```

Comment:
Eventually would want to print out UTC time of when FDP started dp_comp.exe commands. Not necessary, but just a thought to look back at when the commands started.

Reply:
Yep that sounds good we can do that in the next branch